src/read.c

```diff
@@ -1737,19 +1737,6 @@ void ReadLiteral (
         ReadRecExpr( follow );
         break;
 
-#ifdef HPCGAP
-    /* `Literal                                                            */
-    case S_BACKQUOTE:
-        Match( S_BACKQUOTE, "`", follow );
-        TRY_READ {
-          IntrRefGVar(GVarName("MakeImmutable"));
-          IntrFuncCallBegin();
-        }
-        ReadAtom( follow, 'r' );
-        TRY_READ { IntrFuncCallEnd(1, 0, 1); }
-        break;
-#endif
-
```

Comment:
My only thought, is there enough hpc-gap code out there that we should put a SyntaxError here telling people that ` should be rewritten as MakeImmutable?